components/scream/data/scream_input.yaml

```diff
@@ -61,31 +77,32 @@ HOMME:
     partmethod        : 4
     cubed_sphere_map  : 0
     topology          : cube
-    test_case         : jw_baroclinic
     u_perturb         : 1
     rotate_grid       : 0
     ne                : 4
     statefreq         : 9999
     mesh_file         : /dev/null
     tstep             : 300
-    nu                : 7.0e+15
-    nu_div            : 1.0e+15
-    nu_p              : 7.0e+15
-    nu_q              : 7.0e+15
-    nu_s              : -1
-    nu_top            : 2.5e+5
+    nu                :  .000000034
+    nu_div            : -1 
+    nu_p              : .000000034
```

Comment:
These can be `-1` to adopt values from `nu`.

Reply:
You would think, but when I do that I get an error with the nu values being -1.  So I think our namelist parser still has some kinks.